### ISSUE TITLE
Is this loss normal?

### ISSUE THREAD

**Issue by Lali-Lali** [SimonZeng7108/efficientsam3#20]
I'm training the image encoder in Stage 1 using the es_rv_m.yaml configuration, and the training loss is as high as over 1000. Is this a normal phenomenon? 

At the same time, when I load ImageNet pretrained weights for repvit_m1_1, the loss remains similarly high, which puzzles me.

<img width="1916" height="290" alt="Image" src="https://github.com/user-attachments/assets/5da8e862-6120-416a-887d-5c60e1b729bb" />

**Comment by Lali-Lali** [comment]
Looking forward to your reply. Thank you.

**Comment by ShuaiWang97** [comment]
Hi, we also observed a relatively high loss during Stage 1 fine-tuning. However, the final fine-tuned model performance was consistent with the reported results. After 50 finetune epochs on repvit_m1_1, the training loss was still above 1200, but it did not seem to negatively affect the final performance.

**Comment by ShuaiWang97** [comment]
Please feel free to try other models as well.

**Comment by Lali-Lali** [comment]
Another question: During training, it's acceptable if grad_norm occasionally appears as inf, right?    Looking forward to your reply. Thank you.

**Comment by ShuaiWang97** [comment]
grad_norm might occasionally appears as inf at beginning but later should be more stable

**Comment by Lali-Lali** [comment]
Cool, thank you so much. This project has been as helpful as a mentor to me. Looking forward to the updates for Stage 2 and Stage 3.
